org.ppsspp.PPSSPP.yml

```diff
@@ -51,3 +55,26 @@ modules:
 
       - type: file
         path: org.ppsspp.PPSSPP.metainfo.xml
+
+  - name: ppsspp-localization
+    buildsystem: simple
+    build-commands:
+      - |
+        for LANG_FILE in assets/lang/*.ini
+          do ln -fsr "$FLATPAK_DEST/share/locale/$(basename "$LANG_FILE" | grep -Eo '^[a-z]{2}')/ppsspp/$(basename "$LANG_FILE")" "$FLATPAK_DEST/share/ppsspp/$LANG_FILE"
+          install -Dm644 "$LANG_FILE" "$FLATPAK_DEST/share/locale/$(basename "$LANG_FILE" | grep -Eo '^[a-z]{2}')/ppsspp/$(basename "$LANG_FILE")"
+        done
+    sources:
+      - type: git
+        url: https://github.com/hrydgard/ppsspp.git
+        tag: v1.13.2
+        commit: 9fe6338e3bf397f8a009a51a282c139dfa180eb6
+        disable-submodules: true
```

Comment:
Better to use shell source.  
```
- type: shell
  commands:
  - cp -a "$FLATPAK_DEST/share/ppsspp/assets" .
```

Reply:
Nah, I do not think so. Better work on virgin sources in every `module`. Besides, pulling in this `git` `source` costs almost nothing because it has been already cached when processing the `ppsspp` `module`. Furthermore, this way we do not have to keep track of the state of the source directory from `module` to `module`. This will especially become apparent should we add another `module` to the manifest.

Reply:
I don't want to change two times of the manifest every update.   
If the files that this module required is already in `$FLATPAK_DEST`, copying them is the better choice.  
If not, better to write commands to `ppsspp` module directly.

Reply:
Look, the beauty of this approach is that it is automated, accounts for new or dropped l10ns from upstream, and you do not have to touch the manifest twice. But of course, you are free to do as you see fit. Imho, having a dedicated `module` that installs from the exact same source gives a far cleaner, more legible and more robust solution.

Reply:
> the beauty of this approach is that it is automated

The source's git tag is needed update and keep same as the `ppsspp` module.  
If there are more module like this, I need to manually modify them every update.  
Or we add `x-checker-data` to every module like this for the same the source, I don't think it a good idea.

Reply:
Hmm, I see. I think I may have solved your issue with YAML’s references feature. Please see if it works for you.